hoodie-integ-test/pom.xml

```diff
@@ -6,35 +6,62 @@
     <version>0.4.8-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <artifactId>hoodie-integ-test</artifactId>
   <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>hoodie-integ-test</artifactId>
+
   <dependencies>
-    <dependency>
-      <groupId>org.glassfish.jersey.connectors</groupId>
-      <artifactId>jersey-apache-connector</artifactId>
-      <version>2.17</version>
-    </dependency>
+    <!-- Glassfish -->
+    <!-- needs to be at top -->
     <dependency>
       <groupId>org.glassfish.jersey.core</groupId>
       <artifactId>jersey-server</artifactId>
-      <version>2.17</version>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish.jersey.connectors</groupId>
+      <artifactId>jersey-apache-connector</artifactId>
     </dependency>
     <dependency>
       <groupId>org.glassfish.jersey.containers</groupId>
       <artifactId>jersey-container-servlet-core</artifactId>
-      <version>2.17</version>
     </dependency>
+
+    <!-- Docker-Java - Test-->
+    <!-- needs to be at top to force javax.ws version -->
```

Comment:
can you leave more inline comments like this, on why certain things need to be in that order?

Reply:
There are a lot of dependencies. But in general my approach has been to ensure we get the versions of packages which we specify in the base pom. (Unless there is some issue which requires precedence, in which case I have usually left a comment)

Reply:
again makes sense.